docs/flink-ddl.md

```diff
@@ -159,29 +162,16 @@ Table create commands support the commonly used [Flink create clauses](https://n
 * `COMMENT 'table document'` to set a table description.
```

Comment:
I recommend describe "PRIMARY KEY" here, and add a separate section to explain it, like https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/dev/table/sql/create/#primary-key